server/routes/title.js

```diff
@@ -0,0 +1,25 @@
+const router = require('express').Router();
+
+let titleBanner = require('../../db/title.model');
+
+const dataGeneratorFunction = require('../example.data')
+
+router.route('/get').get((req, res) => {
+  titleBanner.Title.find().sort({_id:-1}).limit(5)
+    .then(data => res.status(200).json(data))
+    .catch(err => res.status(400).json(err))
+});
+
+
+//seeding route
+router.route('/add').post((req, res) => {
+  var titleName = dataGeneratorFunction.exampleData(req.body.total);
+  console.log('------>', titleName)
+  titleBanner.saveTile(titleName, (data, err) => {
+    if(err) res.status(400).json(err);
+    else console.log('Titles added successfully')
+  })
+  res.json('Added title names successfully');
+});
```

Comment:
Mentioned above - this is the section I was referring to where you're already splitting out `/get` and `/add`

Reply:
Awesome tip thanks! Now I have just 1 route in index.js and here I split them with get and add